### PR TITLE
LL87 parser updates

### PR DESCRIPTION
**Pull request by kflemin**

### CODE REVIEW

File: bsyncviewer/lib/usecase_parser.py [hunk 1]
```diff
@@ -64,7 +76,9 @@ def process_usecase(use_case_object):
                     rec = UseCaseAttribute(
                         use_case=use_case_object,
                         attribute=attrib,
-                        state=state_val
+                        state=state_val,
+                        group_id=groupId,
```
Comment (nllong) [inline bsyncviewer/lib/usecase_parser.py]: nice!

File: bsyncviewer/lib/validator/workflow.py [hunk 2]
```diff
@@ -112,62 +112,209 @@ def validate_all(self):
         resp.update(self.validate_use_cases())
         return resp
 
+    # from SEED codebase but doesn't work here
```
Comment (nllong) [inline bsyncviewer/lib/validator/workflow.py]: can you remove this code if it doesn't work? sorry that it doesn't work ;(

File: bsyncviewer/lib/validator/workflow.py [hunk 3]
```diff
@@ -112,62 +112,209 @@ def validate_all(self):
...
         for attr in attrs:
             # try to find it
             paths = copy.deepcopy(attr.attribute.path.split("."))
-            loopingVar = copy.deepcopy(self.xml)
+
             # special case: UDF
+            # print("ATTR PATH: {}".format(attr.attribute.path))
             if 'UserDefinedField.FieldName' in attr.attribute.path:
                 # retrieve all FieldNames from UseCaseUDF
                 udfFields = UseCaseUDF.objects.filter(use_case_attribute=attr)
```
Comment (nllong) [inline bsyncviewer/lib/validator/workflow.py]: your JS syntax is creeping into python :D... udf_fields.

Comment (Myoldmopar) [reply bsyncviewer/lib/validator/workflow.py]: m_whatsWrongWith_this_Syntax

**Comment by nllong** [comment]
Reevaluate this based on #11 and the use of schematron.

File: bsyncviewer/lib/validator/workflow.py (hunk 3 above)
Comment (kflemin) [reply bsyncviewer/lib/validator/workflow.py]: I don't see a problem.

**Pull request merged**